features/consent/use-dm-consent.tsx

```diff
@@ -16,17 +16,17 @@ import {
 import { updateConsentForGroupsForAccount } from "./update-consent-for-groups-for-account";
 import { updateInboxIdsConsentForAccount } from "./update-inbox-ids-consent-for-account";
 
-export function useDmConsent(args: {
-  peerInboxId: InboxId;
-  conversationId: ConversationId;
-  topic: ConversationTopic;
-}) {
-  const { peerInboxId, conversationId, topic } = args;
-
+export function useDmConsent() {
   const currentAccount = useCurrentAccount()!;
 
   return useMutation({
-    mutationFn: async (args: { consent: "allow" | "deny" }) => {
+    mutationFn: async (args: {
```

Comment:
does moving the arguments to the functions instead of the hook fix anything or is it just more idiomatic?

Reply:
I just feel like if we're only using the args when calling the mutation, might as well pass them when we call the mutation.